boards/arm/thingy91_nrf9160/Kconfig.board

```diff
@@ -1,17 +1,17 @@
-# Kconfig - nRF91 PCA20035 board configuration
+# nRF91 THINGY91 board configuration
```

Comment:
Should this line be changed as well to match the `Thingy:91 nRF9160 board configuration` above?